app/models/recipe.rb

```diff
@@ -0,0 +1,17 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods
+  has_many :foods, through: :recipe_foods
+
+  scope :public_recipes, -> { where(public: true) }
+
```

Comment:
- [ ] Please make sure that when a recipe is switched from public to private, it is saved in the database and it should not show on the public recipes page :pray: 
  
  [Screencast from 07-15-2023 08:30:25 PM.webm](https://github.com/TaiwoEnoch/FoodyOnRails/assets/92303891/fe75bb7c-d587-4970-b8ca-815e66d4fcff)